tests/integ/jobmanager/common.go

```diff
@@ -424,3 +424,23 @@ func (suite *TestJobManagerSuite) TestJobManagerJobCancellationFailure() {
 	require.Equal(suite.T(), 1, len(ev))
 
 }
+
+func (suite *TestJobManagerSuite) TestTestStepNoLabel() {
+	go func() {
+		suite.jm.Start(suite.sigs)
+		close(suite.jobManagerCh)
+	}()
+
+	_, err := suite.startJob(jobDescriptorNoLabel)
```

Comment:
Since we are returning `ErrStepLabelIsMandatory`, can we assert error type here?

Reply:
Yeah, I was going to do so. But it (`errors.As`) was implemented only in Go1.13, while this package is checked by Go1.11 (or so) in CI. So I decided not to add an extra dependency for an assert only.

Reply:
Fixed.

Reply:
Mh.. we should also fix the Travis configuration to use go 1.13, integration test are run in a go 1.13 container. Thank you for spotting this.